lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -1662,6 +1664,8 @@ class DedupPass : public DedupBase<DedupPass> {
     if (result.failed())
       return signalPassFailure();
 
+    LLVM_DEBUG(llvm::dbgs() << "FIRRTL Deduplication:\n");
```

Comment:
nit: Can you bring this to the start of `runOnOperation` and use `debugPassHeader(this)` that prints a nice header?

Reply:
Done, also added the corresponding footer at the end.